src/Build/Logging/BinaryLogger/BinaryLogger.cs

```diff
@@ -80,6 +82,9 @@ public enum ProjectImportsCollectionMode
         /// </summary>
         public void Initialize(IEventSource eventSource)
         {
+            _intialTargetOutputLogging = Environment.GetEnvironmentVariable("MSBUILDTARGETOUTPUTLOGGING");
+            _initialLogImports = Environment.GetEnvironmentVariable("MSBUILDLOGIMPORTS");
+
```

Comment:
If you're going to get these values anyway, should we add a check if they're explicitly turned off not to turn them on? So if they're set to `false` or `0` or whatever honor that, otherwise set them normally?

Reply:
We could do this for `MSBUILDLOGIMPORTS` because the enabling code checks for a specific value. However, can't be done for `MSBUILDTARGETOUTPUTLOGGING` because the enabling code checks for not null or empty. We'd need to introduce a new environment variable with an actual value and then keep using the old one for back compat. So I'd say not worth doing.

@KirillOsenkov Would there be any user interest to add the feature to turn off import and target output logging for the binary logger? Such as perf? If so, then it may be worth introducing a new env var which can be turned off.

Reply:
Perhaps, not sure if worth doing it now.